CODE_OF_CONDUCT.md

```
@@ -78,7 +78,7 @@ By signing below, team members agree to follow this Code of Conduct and contribu
 | Name | Signature | Date |
 | --- | --- | --- |
 | Adegbite Raheem | R.A | 31-01-26 |
-| |  |
+| Amidat Abiola Ishola|A.I  |01-02-26
```

markdownlint-cli2 (0.20.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)